tools/ci_build/github/android/default_full_aar_build_settings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "build_abis": [
+        "armeabi-v7a",
+        "arm64-v8a",
+        "x86",
+        "x86_64"
+    ],
+    "android_min_sdk_version": 21,
+    "android_target_sdk_version": 24,
+    "build_params": [
+        "--android",
+        "--parallel",
+        "--cmake_generator=Ninja",
+        "--build_java",
+        "--build_shared_lib",
```

Comment:
We don't need java in this build as it goes into a nuget package.

Reply:
but if it's simpler to also build the java api (e.g. other scripts don't need updating to handle there being no java files) it's fine to leave enabled

Reply:
Without building java, we will need a different script to copy all the `.so` files for different platform, it will be simpler if we use the same settings for different targets